### DISCUSSION TITLE
Axes and NumberPlane Width Are Not Same

### DISCUSSION THREAD

**Discussion by Basavaraj-PN** [3b1b/manim#1832] [Q&A] [answered]
Hi I'm new to Manim, 

I'm not able to make axes width and numberplane width of same.

In the image you can see axis width is less than the number plane.

`axes = Axes()`
`grid = NumberPlane()`
`self.add(axes, grid)`

![asd](https://user-images.githubusercontent.com/23361336/174524001-92b6a2aa-e99f-4eb6-9c02-d02907971864.png)

**Comment by CaftBotti** [comment]
Hi, Basavaraj-PN. They are different, because the NumberPlane uses its own position system. But Axes uses the screen width and height.So they are not the same.

**Comment by CaftBotti** [comment, marked as answer]
Axes uses default settings: width: 14,height: 6.You can change NumberPlane settings to make them OK.
